apollos-church-api/src/data/noPostgres.js

```diff
@@ -0,0 +1,40 @@
+// The purpose of this file is to mock the postgres resolvers to null so the API doesn't fail when
+// there's no database. This will not be necessary when Postgres is required by all Apollos APIs.
+
+export default {
+  resolver: {
+    Query: {
+      followRequests: null,
```

Comment:
I think these might need to be `() => (null)` (in reference to the error you are getting)